rest/src/main/kotlin/builder/guild/VoiceStateModifyBuilder.kt

```diff
@@ -0,0 +1,37 @@
+package dev.kord.rest.builder.guild
+
+import dev.kord.common.entity.Snowflake
+import dev.kord.common.entity.optional.Optional
+import dev.kord.common.entity.optional.OptionalBoolean
+import dev.kord.common.entity.optional.delegate.delegate
+import dev.kord.rest.builder.RequestBuilder
+import dev.kord.rest.json.request.CurrentVoiceStateModifyRequest
+import dev.kord.rest.json.request.VoiceStateModifyRequest
+
+class CurrentVoiceStateModifyBuilder(val channelId: Snowflake) : RequestBuilder<CurrentVoiceStateModifyRequest> {
+
+    private var _requestToSpeakTimestamp: Optional<String> = Optional.Missing()
+
+    private var _suppress: OptionalBoolean = OptionalBoolean.Missing
+
+    var requestToSpeakTimestamp: String? by ::_requestToSpeakTimestamp.delegate()
```

Comment:
This should probably be a date-related object like LocalDateTime.

Also, does the timestamp matter at all? The docs say:
[You are able to set request_to_speak_timestamp to any present or future time.](https://github.com/discord/discord-api-docs/blob/5dcf679ad25ec449bcb1d77e1b113d6f456c11ea/docs/resources/Guild.md#update-current-user-voice-state--patch-guildsguildiddocs_resources_guildguild-objectvoice-statesme)

Can you request to speak at a specific time? Or does any request in the future behave the same? If it's the latter, we could just abstract this.